compiler/rustc_metadata/src/rmeta/mod.rs

```diff
@@ -256,6 +256,7 @@ pub(crate) struct CrateRoot {
     has_alloc_error_handler: bool,
     has_panic_handler: bool,
     has_default_lib_allocator: bool,
+    type_aliases_are_lazy: bool,
```

Comment:
Unfortunately, we don't store the language features used by crate in the metadata and thus I had to modify the crate metadata.

Reply:
Out of curiously, I've tried to use `CrateMetadataRef::get_item_attrs` with `CRATE_DEF_INDEX` but for some reason that just returned an empty iterator. Maybe I'm just missing something.

In any case, looking at `ast::Attribute`s doesn't feel robust or clean to me (e.g. would I need to handle `cfg_attr` myself or would it have already been normalized away by that point?).

Reply:
we erase most attributes cross crate, so that's probably why you are not seeing them